api/src/main/resources/marquez/db/migration/V65__dataset_facets_lineage_event_type_nullable.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE dataset_facets ALTER COLUMN lineage_event_type DROP NOT NULL;
```

Comment:
I'm not entirely convinced we should drop the `NULL` constraint on the `lineage_event_type` column. Would we also want to drop it on `lineage_event_time`? I think it would be odd to have `lineage_event_type` as `NULL`, but `lineage_event_time` with a value (though I completely understand the reasoning behind this). Just feels inconsistent. Thoughts?

Reply:
`event_type` does not describe what it contains and is misleading. But this is how it already is. In fact, the column contains `run_state`. I think it's OK to have nullable `run_state`. 

This is similar problem we had in OL spec 2.0. Although we realised this field does not fill well, we decided it's too much effort to fix it. 

Perhaps we should go with separate PR that renames all `eventType`,`event_type` to  `runState`,`run_state`.